ui/StatusQ/src/statuswindow.cpp

```diff
@@ -19,6 +19,15 @@ StatusWindow::StatusWindow(QWindow *parent)
     });
 }
 
+void StatusWindow::updatePosition()
```

Comment:
How did this appear here?

Reply:
Earlier it was in a header, moved to cpp for consistency. Please note that there were two almost identical pairs of cpp/h in dotherside and sandbox. I merged them taking some parts from one, some from another, preserving the original logic (which was identical in both versions).